### PR TITLE
docs: update the PR checklist

### PR DESCRIPTION
**Pull request by markov00**
## Summary

Checklist is rewritten, adding dark mode + theme changes checks

- [ ] The code changes are specific to a chart type, the chart type label is attached (e.g. `:xy`, `:partition`)
- [ ] The code changes are specific to one or more features, one or more feature labels are attached (e.g. `:interactions`, `:axis`)
- [ ] `Theme` API change is introduced, the `:theme` label is attached and the EUI team is pinged
- [ ] The closing issue/s is connected to a running GH project (if not, recheck with the team the priority)
- [ ] Any new public API export must be added to the `packages/charts/src/index.ts`
- [ ] Unit tests are added/updated to match the most common scenarios
- [ ] The feature requires a specific explanation, documentation and/or a storybook story is added
- [ ] Any added stories must import from `@elastic/charts` except for test data & storybook
- [ ] The code is checked for cross-browser compatibility (Chrome, Firefox, Edge)
- [ ] The visual changes are tested with every available theme (darkmode, lightmode, EUI related)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -44,10 +44,13 @@
...
-- [ ] This was checked for cross-browser compatibility
-- [ ] Proper documentation or storybook story was added for features that require explanation or tutorials
-- [ ] Unit tests were updated or added to match the most common scenarios
+- [ ] The code changes are specific to a chart type, the chart type label is attached (e.g. `:xy`, `:partition`)
+- [ ] The code changes are specific to one or more features, one or more feature labels are attached (e.g. `:interactions`, `:axis`)
+- [ ] `Theme` API change is introduced, the `:theme` label is attached and the EUI team is pinged
+- [ ] The closing issue/s is connected to a running GH project (if not, recheck with the team the priority)
+- [ ] Any new public API export must be added to the `packages/charts/src/index.ts`
+- [ ] Unit tests are added/updated to match the most common scenarios
+- [ ] The feature requires a specific explanation, documentation and/or a storybook story is added
```
Comment (nickofthyme) [inline .github/pull_request_template.md:53]:
```suggestion
- [ ] The proper documentation and/or storybook story has been added or updated
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -44,10 +44,13 @@
...
+- [ ] Any added stories must import from `@elastic/charts` except for test data & storybook
```
Comment (nickofthyme) [inline .github/pull_request_template.md:54]: We could prevent this with linting rules

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -44,10 +44,13 @@
...
+- [ ] The code is checked for cross-browser compatibility (Chrome, Firefox, Edge)
```
Comment (nickofthyme) [inline .github/pull_request_template.md:55]:
```suggestion
- [ ] The code has been checked for cross-browser compatibility (Chrome, Firefox, Safari, Edge)
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -44,10 +44,13 @@
...
+- [ ] The visual changes are tested with every available theme (darkmode, lightmode, EUI related)
```
Comment (nickofthyme) [inline .github/pull_request_template.md:56]:
```suggestion
- [ ] Visual changes have been tested with all available themes including `dark`, `light`, `eui-dark` & `eui-light`
```

**Review by nickofthyme — APPROVED** [review]
I added my 2 💶 from middle school english 101. Feel free to expand or change any recommendations I'm not too worried about it.

**Comment by markov00** [comment]
docs: using nick changes

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -44,10 +44,12 @@
...
-- [ ] The proper chart type label was added (e.g. :xy, :partition) if the PR involves a specific chart type
-- [ ] The proper feature label was added (e.g. :interactions, :axis) if the PR involves a specific chart feature
-- [ ] Whenever possible, please check if the closing issue is connected to a running GH project
-- [ ] Any consumer-facing exports were added to `packages/charts/src/index.ts` (and stories only import from `../src` except for test data & storybook)
-- [ ] This was checked for cross-browser compatibility
-- [ ] Proper documentation or storybook story was added for features that require explanation or tutorials
-- [ ] Unit tests were updated or added to match the most common scenarios
+- [ ] The proper **chart type** label has been added (e.g. `:xy`, `:partition`)
+- [ ] The proper **feature** labels have been added (e.g. `:interactions`, `:axis`)
+- [ ] The `:theme` label has been added and the @elastic/eui-design team has been pinged when there are `Theme` API changes
```
Comment (cchaos) [inline .github/pull_request_template.md:49]: Using our handle in here is now actually going to ping us on every PR 😬

Comment (nickofthyme) [reply .github/pull_request_template.md:49]: Oops 🤦🏼 , fixed in https://github.com/elastic/elastic-charts/commit/b4bb0641e5902e4b40863bd7eb5c2b66109e3617

**Pull request merged**